General/output.md

```diff
@@ -4,11 +4,28 @@
 
 - **log/**  - contains  `idea.log`, `gradle.log`, and so on.
 - **projectStructure/** - Metainformation about your project: modules, frameworks/libraries, roots, and so on.
+- **report/** - HTML report (generated via `--save-report`/`--show-report`)
 - **profile.xml** - effective profile in the IDEA format: all inspections and their configurations. Depends on the configured profile, `qodana.yaml`, and the plugin list.
 - **.descriptions.json** - descriptions for all inspections.
 - **%InspectionId%.json** - contains all problems generated by the inspection with this ID: `%InspectionId%` message, place, severity, and so on.
 
 
 ## UI-compatible output
 
-[TO DO]
\ No newline at end of file
+In addition to programmatic output you can also generate human readable output in HTML format (`--save-report`). Unfortunately due to js security restrictions this cannot be currently viewed via file:// protocol (double-click on index.html). For convenience, we provide `--show-report` argument to serve HTML report via http:// protol locally.  
+In case you want to view UI in already generated `report/` folder:
+ - Docker
+    ```
+    docker run -it --rm -p 8000:80 -v $(pwd)/report:/usr/share/nginx/html nginx
+    ```  
+ - Python  
+    If you have python installed, you can serve current folder content via:
+    ```
+    cd report/
+    python2 -m SimpleHTTPServer
+    # or
+    python3 -m http.server
+    ```
+Report would be available at [http://localhost:8000](http://localhost:8000), you can stop the web server by pressing `Ctrl-C`.
+
+In case of [GitLab CI](https://github.com/JetBrains/Qodana/tree/main/Docker#quick-start-with-recommended-profile) you can directly view the report by opening `report/index.html` from artifacts. For [Github Action](https://github.com/JetBrains/Qodana/tree/main/Docker#quick-start-with-recommended-profile) consider uploading artifacts to some [s3 bucket](https://docs.aws.amazon.com/AmazonS3/latest/user-guide/static-website-hosting.html) or using [Qodana Github App](https://github.com/JetBrains/Qodana/blob/main/GitHub/README.md#qodana-github-app) instead.
```

Comment:
```suggestion
### GitLab CI

In case of [GitLab CI](https://github.com/JetBrains/Qodana/tree/main/Docker#quick-start-with-recommended-profile), you can directly view the report by opening `report/index.html` from artifacts. 

### Github action

For [Github Action](https://github.com/JetBrains/Qodana/tree/main/Docker#quick-start-with-recommended-profile), consider uploading artifacts to some [s3 bucket](https://docs.aws.amazon.com/AmazonS3/latest/user-guide/static-website-hosting.html) or using [Qodana Github App](https://github.com/JetBrains/Qodana/blob/main/GitHub/README.md#qodana-github-app) instead.
```